samples/cpp/benchmark_app/main.cpp

```diff
@@ -58,6 +58,9 @@ bool parse_and_check_command_line(int argc, char* argv[]) {
         show_usage();
         throw std::logic_error("The percentile value is incorrect. The applicable values range is [1, 100].");
     }
+    if (FLAGS_api == "") {
```

Comment:
Please also update the logic for benchmark_app.py

Reply:
Updated.